src/styles/reset.css

```diff
@@ -0,0 +1,7 @@
+body,
+p,
+a,
+h2 {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Actually, all of this elements don't have any default paddings, so only reset margins. (links don't have both default margins and paddings)